lib/std/zig/parser_test.zig

```diff
@@ -2885,20 +2964,20 @@ test "zig fmt: multiline string in array" {
     try testCanonical(
         \\const Foo = [][]const u8{
         \\    \\aaa
-        \\,
+        \\    ,
         \\    \\bbb
         \\};
         \\
         \\fn bar() void {
         \\    const Foo = [][]const u8{
         \\        \\aaa
-        \\    ,
+        \\        ,
```

Comment:
This looks inconsistent with the newly added test `test "zig fmt: Only indent multiline string literals in function calls"`

Reply:
Yeah, this is an awkward situation. Everything gets indented inside the `{}` in the above, but in the "Only indent" test nothing gets indented. Because there's no trailing space it's kind-of a pseudo-single-line function call, so a special case is made for multiline strings. If #5343 they could trivially be made to both indent the comma by simply indenting everything in the function call (NB: this would shift in the comma in all testTransforms). To unindent the above comma, as there would be a special case for the comma after a multiline string in an array init.

Reply:
At this point, you've invested enough time and thought on the subject that I would trust your judgement. So this will be your call.